### ISSUE TITLE
[Question] Can I change default javadoc value of the class?

### ISSUE THREAD

**Issue by nqminhuit** [redhat-developer/vscode-java#1243]
Every time I generate a new class in java, the default javadoc is just a repeated of the class name:
```java
/**
 * NewClass
 */
public class NewClass {

}
```
Can I change this value, e.g. author name
```java
/**
 * @author <author_name>
 */
public class NewClass {

}
```
or completely remove it?

Thank you for your time!

**Comment by nqminhuit** [comment]
Hello,

Could anyone provide me a hint on this issue?

Many thanks

**Comment by NikolasKomonen** [comment]
If I understand correctly, you want to add 'author' into your javadoc?

Yes you can, it is optional and will not be an issue if you remove it.

**Comment by nqminhuit** [comment]
No, I mean the default javadoc when generating new class, we don't want to have the class name in javadoc by default instead we need the author by default, not to have every time delete the class name in javadoc and re-enter the author.

**Comment by fbricon** [comment]
This was fixed in 0.67.0. See https://github.com/redhat-developer/vscode-java/issues/1470#issuecomment-692974594
